frontend/talentsearch/src/js/components/Home/partials/CallToAction/CallToAction.tsx

```diff
@@ -0,0 +1,168 @@
+// Vendor dependencies
+import React from "react";
+
+// Local helper dependencies
+
+// Local component dependencies
+
+// Define the interface
+export interface CallToActionProps {
+  type: string;
+  context: string;
+  content: {
+    path: string;
+    label: string;
+  };
+}
+
+// Create the page component
+const CallToAction: React.FC<CallToActionProps> = ({
+  type,
+  context,
+  content,
+}): React.ReactElement => {
+  let buttonStyles = {};
+  let icon;
+  if (context === "job") {
+    buttonStyles = {
+      "data-h2-background-color":
+        "base(white) base:children[div:first-child](tm-yellow) base:children[div:first-child]:hover(tm-yellow.lighter) base:focus-visible(focus) base:focus-visible:children[div:not(:first-child)](focus)",
+      "data-h2-border":
+        "base:children[div:first-child](all, 3px, solid, tm-yellow) base:children[div:not(:first-child)](all, 3px, solid, white) base:focus-visible:children[div:not(:first-child)](all, 3px, solid, focus)",
+      "data-h2-color": "base:children[*](black)",
+    };
+    icon = (
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+        strokeWidth="2"
+        stroke="currentColor"
+        className="w-6 h-6"
+      >
+        <path
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z"
+        />
+      </svg>
```

Comment:
Extract as icon component?

Reply:
Moved the icons to their own components and changed the logic to use our map pattern.

[move cta icons to components](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4049/commits/3a8816672ba7070fe2159aa3d0149c4040cf61ac)